ably-ios/ARTAuth.m

```diff
@@ -84,8 +86,26 @@ - (ARTAuthOptions *)mergeOptions:(ARTAuthOptions *)customOptions {
     return customOptions ? [self.options mergeWith:customOptions] : self.options;
 }
 
+- (void)storeOptions:(ARTAuthOptions *)customOptions {
```

Comment:
But it should not store all the elements and `copyWithZone:` should copy all of them as good practice.